firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [feature] You can now query across all collections in your database with a
```

Comment:
AFAIU, it's only possible to query the most nested collection (e.g., in `col1/doc/col2`, the only suitable collection is `col2`). Is it something that should be clear without additional explanation, or should we call it out somehow?